src/pages/new/volunteer/index.js

```diff
@@ -57,6 +61,38 @@ export default () => (
         },
       ]}
     />
+    <ThreeParagraphBlock
+      title="Let’s work together"
+      paragraph1={{
+        title: "Technology",
+        description:
+          "Being based out of San Francisco, tech professionals comprise the bulk of our volunteer base and work together to improve the digital infrastructure powering the services designated for underserved residents.",
+      }}
+      paragraph2={{
+        title: "Partnerships & Communications",
+        description:
+          "We work with government agencies, nonprofits, and local businesses to strengthen the city’s social services ecosystem. via discourse and knowledge sharing between organizations that align with our mission.",
+      }}
+      paragraph3={{
+        title: "Research & Community Development",
+        description:
+          "To better understand the specific experiences people have endured with housing insecurity, our research team works alongside Community Representatives to better understand how best to serve these challenges.",
+      }}
+      leftTopImage={{
+        url: pairProgrammingImage,
+        alt: "Multiple volunteers working at a datathon.",
+      }}
+      leftBottomImage={{
+        url: teamActivityImage,
+        alt: "Team posing for a photo after a design workshop.",
+      }}
+      rightImage={{
+        url: teamSnackingImage,
+        alt: "Multiple volunteers working at a datathon.",
+      }}
```

Comment:
I think the top left and bottom right images are swapped, except I'm not sure if the file names are swapped or if the code is swapped.

Reply:
You're right! I also need to change the `alt` text. Thanks for catching this!